docs/source/distributed.rst

```diff
@@ -395,6 +395,24 @@ of 16
 .. autofunction:: all_gather_multigpu
 
 
+Third-party backends
+--------------------
+
```

Comment:
Shall we mark this as experimental for now? You can add sth like:

```
.. warning::
  The third-party backend API is experimental and subject to change.
```

Reply:
will fix it